RHEL/7/input/oval/service_nails_enabled.xml

```diff
@@ -0,0 +1,99 @@
+<def-group>
+ <!-- THIS FILE IS GENERATED by create_services_enabled.py.  DO NOT EDIT.  -->
+  <definition class="compliance" id="service_nails_enabled"
+  version="1">
+    <metadata>
+      <title>Service nails Enabled</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 6</platform>
+      </affected>
+      <description>The nails service should be enabled if possible.</description>
+    </metadata>
+    <criteria comment="package nails installed and service nails is configured to start" operator="AND">
+    <extend_definition comment="nails installed" definition_ref="package_nails_installed" />
```

Comment:
Another issue being the ```package_nails_installed``` criterion isn't defined (but that's the issue of the template). Will fix this during manual merge too.